### PR TITLE
Visa card, data faker

### PR DESCRIPTION
**Pull request by jharoonalishah**
## Overview
An easy way to generate fake data for Visa card.

## Update files
- README
- Faker.java
- EnFile.java

## New file
- Visa.java
- VisaTest.java

### CODE REVIEW

File: src/main/java/net/datafaker/Visa.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        super(faker);
+    }
+
+    public String cardNumber() { return faker.fakeValuesService().resolve("visa.cardNumber", this);}
+
+    public String cardName() { return faker.fakeValuesService().resolve("visa.cardName", this);}
+
+    public String expiryDate() { return faker.fakeValuesService().resolve("visa.expiryDate", this);}
+
+    public String csv() { return faker.fakeValuesService().resolve("visa.csv", this);}
```
Comment (snuyanzin) [inline src/main/java/net/datafaker/Visa.java:17]: Please fix code style like it is done in other classes e.g. https://github.com/datafaker-net/datafaker/blob/master/src/main/java/net/datafaker/StarWars.java

File: src/main/resources/en/visa.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+en:
+  faker:
+    visa:
+      cardNumber: ["4111222233334444", "4222333344445555", "4333444455556666", "4777111122223333", "4888444422221111", "4888000011112222", "4444555566667777", "4666444433331111"]
```
Comment (snuyanzin) [inline src/main/resources/en/visa.yml:4]: We have number generators, why can't we use it here?

File: src/main/resources/en/visa.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+      cardName: ["Khan", "Ali", "Khalid", "Khabib", "John", "Perry", "Olivia", "Emma", "Charlotte","Yuki", "Suzuki San", "Toyoda San", "Yui","Khamzat", "Mia", "Kaori", "Pia", "Janan", "Jahanzeb", "Lal badshah", "Aya", "Emily", "Penelope", "Layla", "Hazel", "Madison", "James", "Jack", "Michael", "Tony", "Conor", "Hogan", "Daniel", "Cormier", "Ethan","Owen", "Asher", "Wyatt", "Bruce", "Leo", "Luke", "Hudson", "Jacob", "Issac", "Luca", "Gabriel", "Maria", "Kikko", "Hiroko", "Yoko"]
```
Comment (snuyanzin) [inline src/main/resources/en/visa.yml:5]: We have name generators, why can't we use it here?

File: src/main/resources/en/visa.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+      expiryDate: ["01/01", "01/02", "01/03", "01/04", "01/05", "02/01", "02/02", "02/03", "02/04", "02/05", "03/01", "03/02", "03/03", "03/04", "03/05", "01/29", "01/28", "01/27", "02/26", "02/25", "02/24", "03/23", "03/22", "03/20", "04/10", "04/11", "04/12", "04/13", "05/14", "05/15", "06/16", "06/17", "07/18", "07/19", "08/20", "08/21", "08/22", "09/30", "09/29", "10/27", "10/28", "11/01", "11/23", "12/30", "12/29", "12/25", "12/11"]
+      csv: ["123", "134", "111", "335", "235", "984", "109", "333", "456", "874", "983", "154", "342", "199", "101", "783", "983", "385", "389", "501", "404", "202", "303", "606", "666", "777", "888", "311", "731"]
```
Comment (snuyanzin) [inline src/main/resources/en/visa.yml:7]: We have number generators, why can't we use it here?

File: src/main/resources/en/visa.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+en:
+  faker:
+    visa:
+      cardNumber: ["4111222233334444", "4222333344445555", "4333444455556666", "4777111122223333", "4888444422221111", "4888000011112222", "4444555566667777", "4666444433331111"]
+      cardName: ["Khan", "Ali", "Khalid", "Khabib", "John", "Perry", "Olivia", "Emma", "Charlotte","Yuki", "Suzuki San", "Toyoda San", "Yui","Khamzat", "Mia", "Kaori", "Pia", "Janan", "Jahanzeb", "Lal badshah", "Aya", "Emily", "Penelope", "Layla", "Hazel", "Madison", "James", "Jack", "Michael", "Tony", "Conor", "Hogan", "Daniel", "Cormier", "Ethan","Owen", "Asher", "Wyatt", "Bruce", "Leo", "Luke", "Hudson", "Jacob", "Issac", "Luca", "Gabriel", "Maria", "Kikko", "Hiroko", "Yoko"]
+      expiryDate: ["01/01", "01/02", "01/03", "01/04", "01/05", "02/01", "02/02", "02/03", "02/04", "02/05", "03/01", "03/02", "03/03", "03/04", "03/05", "01/29", "01/28", "01/27", "02/26", "02/25", "02/24", "03/23", "03/22", "03/20", "04/10", "04/11", "04/12", "04/13", "05/14", "05/15", "06/16", "06/17", "07/18", "07/19", "08/20", "08/21", "08/22", "09/30", "09/29", "10/27", "10/28", "11/01", "11/23", "12/30", "12/29", "12/25", "12/11"]
```
Comment (snuyanzin) [inline src/main/resources/en/visa.yml:6]: We have number generators, why can't we use it here?

**Comment by bodiam** [comment]
Let's pause this PR for now.

**Pull request closed without merging**